CONTRIBUTING.md

```diff
@@ -104,48 +93,34 @@ conda uninstall pytorch -y
 yes | pip uninstall torch
 ```
 
-2. Clone a copy of PyTorch from source:
-
-```bash
-git clone https://github.com/pytorch/pytorch
-cd pytorch
-```
+* If you want to have no-op incremental rebuilds (which are fast), see [Make no-op build fast](#make-no-op-build-fast) below.
 
-If you already have PyTorch from source, update it:
+* Instead of installing PyTorch via `python setup.py install`, use `python setup.py develop`.
```

Comment:
Arguably, this is already suggested here 
https://github.com/pytorch/pytorch/blob/73e5379fab05c40ff6f42500309cdc17ee57548c/README.md?plain=1#L237
But not removing the details.

Reply:
Nice, I see that since the change made in https://github.com/pytorch/pytorch/pull/88507 this line doesn't even make sense anymore.

It would be great if you could move this bit to there as well

Reply:
I tried elaborating with moving it there, but I realised some of the information is currently duplicated even here

https://github.com/pytorch/pytorch/blob/7bdbef90e99d7f6cd5c1f3a952dbc74c85d8d984/CONTRIBUTING.md?plain=1#L84-L89

https://github.com/pytorch/pytorch/blob/7bdbef90e99d7f6cd5c1f3a952dbc74c85d8d984/CONTRIBUTING.md?plain=1#L113-L116

So I think it's just too much info for `README.md`. It makes more sense to keep various build tips & tricks in this doc. 

I also can't verify if it makes sense anymore, but it feels like it's written by someone who knows and was kind enough to share.

I suggest leaving it here, but I will push a change to merge the two instructions on how to uninstall.

Reply:
Sounds good